docs/designs/loc.md

```diff
@@ -29,6 +29,24 @@ DocFX build supports localization contents, there are a few features need to be
   - Overwrite the configurations you want or use source configuration by default  
   - Focus the Loc publishing configuration controlling, like bilingual or contribution.  
   - Source configuration changing will be immediately applied to Loc, no more manually sync.
+  ```yaml
+  tier1: &TIER1
```

Comment:
Try avoid using JSON incompatible YAML syntax like anchors. It may problematic for future extensions.

Reply:
why it's incompatible, it can be converted to JSON?